<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWidget</class>
 <widget class="QWidget" name="ManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bank</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="cmbBanks"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="btnAddBank">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnRemoveBank">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lblBankInfo">
          <property name="text">
           <string>Bank Information</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="btnBankSettings">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ManagerTreeView" name="assetTree">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QToolButton" name="btnImportAssets">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnImportDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCreateFilter">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnReplaceAssets">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDeleteAssets">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDeleteAssets">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Atlas-Delete.png</normaloff>:/icons16x16/items/Atlas-Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected asset(s). All affected banks will be regenerated.</string>
   </property>
  </action>
  <action name="actionReplaceAssets">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/atlas-replaceFrame.png</normaloff>:/icons16x16/atlas-replaceFrame.png</iconset>
   </property>
   <property name="text">
    <string>Replace Assets</string>
   </property>
   <property name="toolTip">
    <string>Replace the selected assets(s). All affected banks will be regnerated.</string>
   </property>
  </action>
  <action name="actionAddFilter">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Filter-New.png</normaloff>:/icons16x16/items/Filter-New.png</iconset>
   </property>
   <property name="text">
    <string>Add Filter</string>
   </property>
   <property name="toolTip">
    <string>Insert a new filter to organize imported assets</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Renames the selected filter or asset</string>
   </property>
  </action>
  <action name="actionAddBank">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Add Bank</string>
   </property>
   <property name="toolTip">
    <string>Creates a new asset bank</string>
   </property>
  </action>
  <action name="actionRemoveBank">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Bank</string>
   </property>
   <property name="toolTip">
    <string>Removes the currently selected bank.</string>
   </property>
  </action>
  <action name="actionImportAssets">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Atlas-New.png</normaloff>:/icons16x16/items/Atlas-New.png</iconset>
   </property>
   <property name="text">
    <string>Import Assets</string>
   </property>
   <property name="toolTip">
    <string>Import 1 or more assets from a single directory</string>
   </property>
  </action>
  <action name="actionImportDirectory">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Atlas-Open.png</normaloff>:/icons16x16/items/Atlas-Open.png</iconset>
   </property>
   <property name="text">
    <string>Import Asset Directory</string>
   </property>
   <property name="toolTip">
    <string>Imports all assets found recursively from a start directory</string>
   </property>
  </action>
  <action name="actionBankSettings">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-settings.png</normaloff>:/icons16x16/generic-settings.png</iconset>
   </property>
   <property name="text">
    <string>Bank Settings</string>
   </property>
   <property name="toolTip">
    <string>Change asset bank options and for the currently selected bank</string>
   </property>
  </action>
  <action name="actionAssetSettings">
   <property name="icon">
    <iconset resource="../../../data/HyEditor.qrc">
     <normaloff>:/icons16x16/items/Atlas-Settings.png</normaloff>:/icons16x16/items/Atlas-Settings.png</iconset>
   </property>
   <property name="text">
    <string>Asset Properties</string>
   </property>
   <property name="toolTip">
    <string>Configure all selected assets' properties</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ManagerTreeView</class>
   <extends>QTreeView</extends>
   <header>ManagerWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
